deployment/stacks/imports_included/data.ts

```diff
@@ -0,0 +1,93 @@
+import { Construct } from "constructs";
+import {
+  aws_dynamodb as dynamodb,
+  aws_iam as iam,
+  aws_s3 as s3,
+  CfnOutput,
+  Aws,
+  Tags,
+} from "aws-cdk-lib";
+import { DynamoDBTable } from "../../constructs/dynamodb-table";
+
+interface CreateDataComponentsProps {
+  scope: Construct;
+  stage: string;
+  isDev: boolean;
+}
+
+export function createDataComponents(props: CreateDataComponentsProps) {
+  const { scope, stage, isDev } = props;
+
+  const service = "database";
+  const tables = [
+    new DynamoDBTable(scope, "Banner", {
+      stage,
+      isDev,
+      name: "banners",
+      partitionKey: { name: "key", type: dynamodb.AttributeType.STRING },
+    }),
+    new DynamoDBTable(scope, "FormTemplateVersions", {
+      stage,
+      isDev,
+      name: "form-template-versions",
+      partitionKey: { name: "reportType", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "versionNumber", type: dynamodb.AttributeType.NUMBER },
+      lsi: [
+        {
+          indexName: "LastAlteredIndex",
+          sortKey: { name: "lastAltered", type: dynamodb.AttributeType.STRING },
+        },
+        {
+          indexName: "IdIndex",
+          sortKey: { name: "id", type: dynamodb.AttributeType.STRING },
+        },
+        {
+          indexName: "HashIndex",
+          sortKey: { name: "md5Hash", type: dynamodb.AttributeType.STRING },
+        },
+      ],
+    }),
+    new DynamoDBTable(scope, "SarReports", {
+      stage,
+      isDev,
+      name: "sar-reports",
+      partitionKey: { name: "state", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "id", type: dynamodb.AttributeType.STRING },
+    }),
+    new DynamoDBTable(scope, "WpReports", {
+      stage,
+      isDev,
+      name: "wp-reports",
+      partitionKey: { name: "state", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "id", type: dynamodb.AttributeType.STRING },
+    }),
+  ];
+  tables.forEach((table) => Tags.of(table).add("SERVICE", service));
```

Comment:
Do we still need all these tags? For some reason I was thinking you'd told me we don't need these anymore.

Reply:
The imports need them as for the imports to work, the tags need to match as-is.

Reply:
@peoplespete you're right.  Fixed.